### PR TITLE
[Enterprise developer portal] Add documentation for the Get started guides.

### PR DESCRIPTION
**Pull request by slega**
## Description
Add documentation for the Get started guides that will be available in 1.1.0 release of the enterprise developer portal.

## Motivation and Context
Documentation for new feature.

## Preview Link

https://deploy-preview-2211--tyk-docs.netlify.app/docs/nightly/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-get-started-guides-for-api-products/

## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [x] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [x] I have [reviewed the guidelines](../CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](../CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-api-docs.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: "Manage Get started guides"
+date: 2022-02-08
+tags: [""]
+description: ""
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-api-docs.md]: Tags and description are a must

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-api-docs.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+title: "Manage Get started guides"
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-api-docs.md]: are you sure about this title?

Comment (slega) [reply tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-api-docs.md]: Yes. This is how they appear in the live portal. We have two types of docs for API Products: Open API Spec and the Get started guides.

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-api-docs.md (hunk 1 above)
Comment (slega) [reply tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/manage-api-docs.md]: Fixed.

**Review by konrad-sol — APPROVED** [review]

**Review by letzya — CHANGES_REQUESTED** [review]
Sent you PM with comments

**Review by sedkis — APPROVED** [review]
LGTM

**Review by letzya — APPROVED** [review]

**Pull request merged**